index.html

```diff
@@ -3,12 +3,125 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+  <link rel="stylesheet" href="https://use.typekit.net/sbx8kxu.css">
   <link rel="stylesheet" href="style.css">
-  <title>Free thinking Conference: The value of reason, logic, and evidence.</title>
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+<body class="body">
```

Comment:
- [OPTIONAL] One of the main requirements in all the html-css module's projects is to make your design responsive. currently your design is responsive above `768px`. but under `768px` your website gets the exact width of `375px` which is wrong and against responsive design. we avoid giving exact widths to our conatiners at all costs in order to achieve the responsive design. 
- Kindly make sure to remove `375px` width from your body tag and try to make your design responsive under 768px up to 375px which is the lowest mobile width and we dont make it responsive under 375px.
your design at 764px
![jpg](https://user-images.githubusercontent.com/96683843/226126531-a7a1a5ee-16dd-4439-a9c3-b386fdf67a70.jpg)